clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1165,6 +1168,10 @@ raw2trace_t::process_syscall_pt(raw2trace_thread_data_t *tdata, uint64_t syscall
                                   .size = TRACE_MARKER_TYPE_SYSCALL_TRACE_START,
                                   .addr = sysnum };
     entries.push_back(start_entry);
+    // TODO i#5505: When ir2trace starts adding synthesized read/write memrefs for
+    // the kernel trace, change the trace file type from
+    // OFFLINE_FILE_TYPE_KERNEL_SYSCALL_INSTR_ONLY to
+    // OFFLINE_FILE_TYPE_KERNEL_SYSCALLS.
```

Comment:
This comment should be in output.cpp?

Reply:
Copied it there also. I kept it here because it would be more difficult to miss.

Reply:
When we add the memory address synthesis support to ir2trace, I mean.